build-tools-internal/src/main/groovy/elasticsearch.ide.gradle

```diff
@@ -239,20 +240,22 @@ if (providers.systemProperty('idea.active').getOrNull() == 'true') {
  * but before the XML document, e.g. a doctype or comment
  */
 void modifyXml(Object path, Action<? super Node> action, String preface = null) {
-  Node xml = parseXml(path)
-  action.execute(xml)
+  if (project.file(path).exists()) {
```

Comment:
I made this lenient since there seems to be some kind of race condition between when these tasks run and these configuration files exist. There's no good solution for this that I've found but at least doing a refresh or subsequent import will apply the missing config. This is really only applicable to fresh project imports on an empty workspace.